HED-schema-reduced.mediawiki

```diff
@@ -0,0 +1,510 @@
+HED version: v1.4.0-restruct
+
+'''Syntax'''  
+
+HED tags can only be separated by commas. Use semicolons (;) instead of commas in event descriptions since otherwise extra commas could confuse HED parsers.  
+From version 2.2, HED adheres to http://semver.org/ versioning. 
+
+!# start hed 
+
+'''Event''' 
+* Category <nowiki>{required, requireChild, position=1} [This is meant to designate the reason this event was recorded. Every event should have at least one category, but may have multiple categories.] </nowiki> 
+** Sensory presentation <nowiki> [Something that can be perceived.]</nowiki>
+*** Experimental <nowiki>{extensionAllowed} [Primary category for time-locked stimulus events.]</nowiki>
+*** Environmental <nowiki>{extensionAllowed}[An environmental event, for example, a change in experimental context such as walking on dirt versus sidewalk or a loud noise in the next room.]</nowiki> 
+
+** Participant action <nowiki>[Any action engaged in by the participant.]</nowiki>
+*** Response {extensionAllowed} <nowiki>[An action taken by a participant in response to an experimental or environmental stimulus.]</nowiki>
+*** Failure {extensionAllowed} <nowiki>{extensionAllowed} [Situation in which participant acts outside of the constraints of the experiment -- such as driving outside the boundary of a simulation experiment or using equipment incorrectly]</nowiki> 
+*** Incidental {extensionAllowed} <nowiki>[An action such as a yawn or stretch not in response to a sensory presentation.]</nowiki>
+** Data feature <nowiki>{extensionAllowed} [An event derived from the data itself, possibly inserted in a post-hoc fashion.]</nowiki>
+** Experiment 
+*** Control <nowiki>{extensionAllowed} [Information about states and events of the software program that controls the experiment]</nowiki> 
+**** Activity <nowiki>[Experiment-specific actions such as moving a piece in a chess game. Use Attribute/Temporal/Onset and Attribute/Temporal/Offset to specify start and end. Use Attribute/ID/Index/# to designate an identifier of the task.]</nowiki> 
```

Comment:
The meaning of *Activity* category is quite vague to me

Reply:
The categories of Experiment/Control are:

Activity
Initial context
Sequence
Setup
Synchronization

If we eliminate Activity, we what about things that are done to actually control the experiment. They don't fit into the other categories.